tests/platform_tests/test_reboot.py

```diff
@@ -153,54 +153,54 @@ def _power_off_reboot_helper(kwargs):
     @summary: used to parametrized test cases on power_off_delay
     @param kwargs: the delay time between turning off and on the PSU
     """
-    psu_ctrl = kwargs["psu_ctrl"]
-    all_psu = kwargs["all_psu"]
+    pdu_ctrl = kwargs["pdu_ctrl"]
+    all_outlet = kwargs["all_outlet"]
```

Comment:
s/all_outlet/all_outlets/

Reply:
Thanks Joe. Done in the last commit. Retested reboot power cycle.